unified-installer/astra-unified-installer.sh

```diff
@@ -1097,10 +1097,9 @@ get_registry_credentials_from_pull_secret() {
     # 6. Get the first matching entry's `.value.auth`, which would be `B64_ENCODED_USERNAME_PASSWORD` here
     # 7. The `B64_ENCODED_USERNAME_PASSWORD` can then be passed to curl via
     # `-H 'Authorization: Basic B64_ENCODED_USERNAME_PASSWORD'` without having to decode it first.
-    local -r contents="$(kubectl get secret "$pull_secret" -n "$namespace" -o json 2> /dev/null)"
+    local -r contents="$(k8s_get_resource "secret/$pull_secret" "$namespace" "json")"
```

Comment:
You'll see quite a few of these changes; most `kubectl get` commands now instead use `k8s_get_resource` or `k8s_resource_exists`, which actually handles kubectl errors instead of just sending them into the void.